lib/store-init-spec.js

```diff
@@ -0,0 +1,87 @@
+var sinon = require('sinon');
+var proxyquire = require('proxyquire');
+var mediator = require('fh-wfm-mediator/lib/mediator');
+var mockStore = require('./mocks/mockStore.js');
+var mockConf = {
+  persistentStore: true,
+  get: function() {
+    return mockConf.persistentStore;
+  }
+};
+var SelectedStore = function(datasetId) {
+  this.datasetId = datasetId;
+};
+SelectedStore.prototype.list = function() {};
+SelectedStore.prototype.init = function() {};
+SelectedStore.prototype.listen = function() {};
+
+var mockSelectStore = function() {
+  return SelectedStore;
```

Comment:
This SelectedStore needs to be returned as is, as initStore uses this to initialize data store: https://github.com/JameelB/raincatcher-demo-cloud/blob/a563e4b7ca49a6f0b6abdf39f1f9a4b42bf24a98/lib/store-init.js#L12